test/test_validation.py

```diff
@@ -583,3 +583,14 @@ def test_custom_message(self):
                 ]
         ), Column('')):
             self.assertRegex(error.message, self.message, 'Validator not using the custom warning message!')
+
+    def test_validating_empty_range_value(self):
```

Comment:
This test should be in a different class, probably its own class. It certainly has nothing to do with `CustomMessage`.

Also, you should assert something meaningful here; check that it produces the actual results you want, rather than just copying the assertion from above.